charts/ehrenamt-justiz/Chart.yaml

```
@@ -0,0 +1,17 @@
+description: Helm Chart for deploying a it@M Ehrenamt-Jusitz application.
+type: application
+version: 0.0.1
+home: https://github.com/it-at-m/helm-charts/tree/main/charts/ehrenamt-justiz
+icon: https://assets.muenchen.de/logos/itm/itm-logo-256.png
+dependencies:
+ - name: refarch-gateway
+   condition: refarch-gateway.enable
+   version: 1.6.1
+   repository: "@it-at-m"
+sources:
+  - "https://github.com/it-at-m/helm-charts"
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 1 but found 2

(indentation)